### DISCUSSION TITLE
Development Contribution/Partnership

### DISCUSSION THREAD

**Discussion by lehoang318** [autoas/ssas-public#19] [Q&A]
I would like to ask whether your team is opened with external contribution?
And if it is possible, I want to know your motivation & vision about the work as well.

Since Arctic Core (which is probably dropped), I'm really glad to see another OSS implementation of AUTOSAR.
Thank you for your effort.

**Comment by parai** [comment]
Contribution is welcome, but please note that this is not a free project for commercial purpose though open-source code, so any external contribution will be used by me for commercial purpose, and you maybe not get paid for that contribution.
